src/backend/web/routes/feeds.js

```diff
@@ -32,7 +32,7 @@ feeds.get('/', async (req, res) => {
   );
 });
 
-feeds.get('/:id', async (req, res) => {
+feeds.get('/:id', validateFeedsIdParam(), async (req, res) => {
```

Comment:
I'm unclear why you've had to add this for `get` as well if you're focused on fixing `delete`?

Reply:
I'm not sure why I did that, I must have got mixed up somewhere, I'll revert it

Reply:
I think what happened here, was that 62a6022 had that change, and for whatever reason, my commit does too
![62a6022](https://user-images.githubusercontent.com/32021785/99568575-99b03b80-299d-11eb-9deb-58637240b4d0.png) possibly due to the rebasing? By reverting it I'm breaking the tests.